app/src/main/java/com/yacgroup/yacguide/AboutActivity.kt

```diff
@@ -82,16 +82,15 @@ class AboutActivity : BaseNavigationActivity() {
 
     private fun _selectContactConcern() {
         val contactInterface = ContactUtils(this)
-        val dialog = AlertDialog.Builder(this).apply {
-            setTitle(getString(R.string.contact_concern))
+        val builder = DialogWidgetBuilder(this, R.string.contact_concern).apply {
             setItems(R.array.contactConcerns) { _, which ->
                 when (which) {
                     0 -> contactInterface.sendFeedback()
                     1 -> contactInterface.sendBugReport()
                 }
             }
-            setNegativeButton(R.string.cancel) { dialog, _ -> dialog.dismiss()}
+            setNegativeButton()
         }
-        dialog.show()
+        builder.show()
```

Comment:
I'm a little bit confused. In some cases you call `show()` on a dialog (created by `DialogWidgetBuilder(...).create()`). But here and in some other code spots you call `show()` on the `DialogWidgetBuilder` object itself. Is this intended? If yes, ignore this and just merge.

Reply:
I went through all the dialog code again and did some further optimization. Normally, I use `show()` of the builder instance. Which I think, is the intended way. However, in some special cases I have to do it a bit different. In these cases, there should be a comment for the reason.